### DISCUSSION TITLE
Out-of-distribution detection in medical imagery

### DISCUSSION THREAD

**Discussion by vfdev-5** [Project-MONAI/MONAI#6634] [Q&A]
I was wondering if MONAI project is planning or already providing out-of-the-box tools for detecting out-of-distribution samples.

I admit that this topic is rather large and can have different kind of implementations: from agnostic to medical domain to something more specific with pretrained model (maybe). So, my question is more about discussing about this topic.
Thank you all for you feedback.

cc @AlexanderChaptykov

**Comment by AlexanderChaptykov** [comment]
@vfdev-5 thanks! Nice idea to notice MONAI.
I will check it
